ethpm_cli/install.py

```diff
@@ -0,0 +1,104 @@
+from argparse import Namespace
+import json
+from pathlib import Path
+import shutil
+import tempfile
+
+from eth_utils import to_text
+from eth_utils.toolz import assoc
+from ethpm.backends.ipfs import BaseIPFSBackend
+from ethpm.utils.ipfs import is_ipfs_uri
+
+from ethpm_cli._utils.ipfs import get_ipfs_backend
+from ethpm_cli.exceptions import InstallError
+from ethpm_cli.package import Package
+from ethpm_cli.validation import validate_parent_directory
+
+
+class Config:
+    """
+    Class to manager CLI config options
+    - IPFS Backend
+    - Target ethpm_dir
+    """
+
+    def __init__(self, args: Namespace) -> None:
+        self.ipfs_backend = get_ipfs_backend(args.local_ipfs)
+        if args.ethpm_dir is None:
+            self.ethpm_dir = Path.cwd() / "ethpm_packages"
+            self.ethpm_dir.mkdir()
+        else:
+            self.ethpm_dir = args.ethpm_dir
+
+
+def install_package(pkg: Package, config: Config) -> None:
+    if (config.ethpm_dir / pkg.alias).is_dir():
```

Comment:
Would it be more appropriate to just do an `exists` check here? since any name conflict is likely an error case?